system/src/Grav/Common/Twig.php

```diff
@@ -183,6 +183,10 @@ public function processPage(Page $item, $content = null)
                 $twig_vars['content'] = $content;
                 $template = $item->template() . TEMPLATE_EXT;
                 $output = $local_twig->render($template, $twig_vars);
```

Comment:
This method need when you use a modular template with more one module. If you compile only one time, you compile only the first module, then other modules is empty.